core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -94,24 +95,116 @@ oppia.directive('topNavigationBar', [function() {
         }
 
         $scope.windowIsNarrow = windowDimensionsService.isWindowNarrow();
+        var currentWindowWidth = windowDimensionsService.getWidth();
+        $scope.navElementsVisibilityStatus = {
+          I18N_TOPNAV_DONATE: true,
+          I18N_TOPNAV_ABOUT: true,
+          I18N_CREATE_EXPLORATION_CREATE: true,
+          I18N_TOPNAV_LIBRARY: true
+        };
+        // The order of the elements in this array specifies the order in which
+        // they will be hidden.
+        navElementsOrder = ['I18N_TOPNAV_DONATE', 'I18N_TOPNAV_ABOUT',
+          'I18N_CREATE_EXPLORATION_CREATE', 'I18N_TOPNAV_LIBRARY'];
+
         windowDimensionsService.registerOnResizeHook(function() {
           $scope.windowIsNarrow = windowDimensionsService.isWindowNarrow();
           $scope.$apply();
           // Close the sidebar, if necessary.
           SidebarStatusService.closeSidebar();
           $scope.sidebarIsShown = SidebarStatusService.isSidebarShown();
-        });
-        $scope.isSidebarShown = function() {
-          if (SidebarStatusService.isSidebarShown()) {
-            angular.element(document.body).addClass('oppia-stop-scroll');
-          } else {
-            angular.element(document.body).removeClass('oppia-stop-scroll');
+          $scope.isSidebarShown = function() {
```

Comment:
This was supposed to be part of the merge, but I did it incorrectly. I compared to https://github.com/oppia/oppia/commit/b9c43c93aaee18b59ec18774381112fe6f998846 and I think I've corrected the merge.